statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -146,39 +146,42 @@ func (wr *WidgetsRender) prepareInstanceInfo() {
 	countConsensusAcceptedBlocks := wr.presenter.GetCountConsensusAcceptedBlocks()
 
 	if shardId == uint64(core.MetachainShardId) {
-		consensusInfo = fmt.Sprintf("Count consensus participant: %d | Signed blocks headers: %d", countConsensus, countConsensusAcceptedBlocks)
-
+		consensusInfo = fmt.Sprintf("Validator signed meta blocks: %d | Signed blocks headers: %d", countConsensus, countConsensusAcceptedBlocks)
 	} else {
-		consensusInfo = fmt.Sprintf("Consensus accepted / signed blocks: %d / %d", countConsensusAcceptedBlocks, countConsensus)
+		consensusInfo = fmt.Sprintf("Validator signed blocks %d | Number accepted: %d", countConsensus, countConsensusAcceptedBlocks)
 	}
 
 	rows[3] = []string{consensusInfo}
 
 	countLeader := wr.presenter.GetCountLeader()
 	countAcceptedBlocks := wr.presenter.GetCountAcceptedBlocks()
-	rows[4] = []string{fmt.Sprintf("Blocks accepted / blocks proposed:  %d / %d", countAcceptedBlocks, countLeader)}
-
-	switch instanceType {
-	case string(core.NodeTypeValidator):
-		rewardsPerHour := wr.presenter.CalculateRewardsPerHour()
-		rows[5] = []string{fmt.Sprintf("Rewards estimation: %s ERD/h (without fees)", rewardsPerHour)}
-
-		var rewardsInfo []string
-		totalRewardsValue, diffRewards := wr.presenter.GetTotalRewardsValue()
-		zeroString := "0" + wr.presenter.GetZeros()
-		if diffRewards != zeroString {
-			wr.instanceInfo.RowStyles[7] = ui.NewStyle(ui.ColorGreen)
-			rewardsInfo = []string{fmt.Sprintf("Total rewards %s + %s ERD (without fees)", totalRewardsValue, diffRewards)}
-		} else {
-			wr.instanceInfo.RowStyles[7] = ui.NewStyle(ui.ColorWhite)
-			rewardsInfo = []string{fmt.Sprintf("Total rewards %s ERD (without fees)", totalRewardsValue)}
-		}
-		rows[6] = rewardsInfo
-
-	default:
-		rows[5] = []string{""}
-		rows[6] = []string{""}
-	}
+	rows[4] = []string{fmt.Sprintf("Blocks proposed: %d | Number accepted:  %d", countLeader, countAcceptedBlocks)}
```

Comment:
This is still not very clear:
rows[4] = []string{fmt.Sprintf("Blocks proposed: %d | Blocks accepted:  %d", countLeader, countAcceptedBlocks)}

Reply:
changed `Number` to `Blocks`